.github/workflows/publish.yaml

```
@@ -0,0 +1,81 @@
+# Publish PyPI and NPM artifacts
+name: Publish
+on:
+  push:
+  pull_request:
```

Providing a manual run button for those with push rights, something I've found generally useful for example to verify that something isn't broken on master that doesn't work in a PR I'm reviewing.

```suggestion
  pull_request:
  workflow_dispatch:
```